### DISCUSSION TITLE
github-merge-queue bot removes pull request from the merge queue due to no response for status checks

### DISCUSSION THREAD

**Discussion by dasmanas** [community/community#47293] [Pull Requests] [answered]
### Select Topic Area

Product Feedback

### Body

github-merge-queue bot removes pull request from the merge queue due to no response for status checks even though when all the status checks are completed as passed.
A reference pull request is https://github.com/pyrsia/pyrsia/pull/1605 which is getting impacted and thrown out of the queue for multiple times.

**Comment by mikocot** [comment]
we've seen the same behavior, have you found the reason for it? It seems that it only happens when the request takes a lot of time, maybe there is some timeout that should be configurable/extended?

**Comment by mikocot** [comment, marked as answer]
Aaaand I answered my own question, there is on in the branch settings :) Hope it helps anyone else who gets here for that very reason

![image](https://github.com/community/community/assets/9884950/97f0812f-44c5-44bc-a654-12f1d85ab7a2)
